src/components/button/button.style.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/prefer-default-export */
+import styled from 'styled-components/macro';
+
+export const ButtonStyle = styled.button`
+  width: ${(props) => (props.size === 'lg' ? '216px' : '92px')};
```

Comment:
Instead of settting a fixed width you might be able to set a common padding. Imagine that you use the button in more places with different text content. Setting a fixed width for every one of those wouldn't be scalable. Another scenario are multi-language pages. Depending on the user language the button's text would change and with it its width. The example implementation uses padding iirc.

Reply:
yeah, a common padding value will be more practical in terms of buttons.